backend/telemetry_core/src/aggregator/aggregator.rs

```diff
@@ -92,6 +92,20 @@ impl Aggregator {
                 send_node_data,
             ));
         } else {
+            tokio::task::spawn({
+                let tx_to_aggregator = tx_to_aggregator.clone();
+                let mut timer = tokio::time::interval(std::time::Duration::from_secs(10));
```

Comment:
Should should be something like 1 second instead. Block production happens every 6 seconds, time slot is 1 second. Sending every 10 seconds would mean we don't see new block produced for quite a while.

Reply:
I guess we can supply that from cli then :)